seatunnel-connectors-v2/connector-doris/src/test/java/org/apache/seatunnel/connectors/doris/serialize/SeaTunnelRowConverterTest.java

```diff
@@ -1,54 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.seatunnel.connectors.doris.serialize;
-
-import org.apache.seatunnel.api.table.type.LocalTimeType;
-
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.Test;
-
-import java.time.LocalDateTime;
-
-public class SeaTunnelRowConverterTest {
-
-    private static final SeaTunnelRowConverter seaTunnelRowConverter = new SeaTunnelRowConverter();
-
-    @Test
-    void testDateTimeWithNano() {
-        Assertions.assertEquals(
-                "2021-01-01 00:00:00.123456",
-                seaTunnelRowConverter.convert(
-                        LocalTimeType.LOCAL_DATE_TIME_TYPE,
-                        LocalDateTime.of(2021, 1, 1, 0, 0, 0, 123456789)));
-        Assertions.assertEquals(
-                "2021-01-01 00:00:00.000000",
-                seaTunnelRowConverter.convert(
-                        LocalTimeType.LOCAL_DATE_TIME_TYPE,
-                        LocalDateTime.of(2021, 1, 1, 0, 0, 0, 0)));
-        Assertions.assertEquals(
-                "2021-01-01 00:00:00.000001",
-                seaTunnelRowConverter.convert(
-                        LocalTimeType.LOCAL_DATE_TIME_TYPE,
-                        LocalDateTime.of(2021, 1, 1, 0, 0, 0, 1000)));
-        Assertions.assertEquals(
-                "2021-01-01 00:00:00.000123",
-                seaTunnelRowConverter.convert(
-                        LocalTimeType.LOCAL_DATE_TIME_TYPE,
-                        LocalDateTime.of(2021, 1, 1, 0, 0, 0, 123456)));
-    }
```

Comment:
we should make sure the behavior not change. Please do not delete this test case. Instand of cover it by `SeaTunnelRowSerializer`.

Reply:
> we should make sure the behavior not change. Please do not delete this test case. Instand of cover it by `SeaTunnelRowSerializer`.

We use seatunnel's serialization tool, so `SeaTunnelRowConverter.java` was deleted. This test case will no longer be used.

Reply:
We need all datetime value written to doris to have the same precision.